xml/depl_maintenance.xml

```diff
@@ -2174,6 +2174,12 @@ migrate</screen>
        Or take the cluster out of maintenance mode.
       </para>
      </step>
+     <step>
+      <para>
+       If necessary, boot the compute plane with the instructions in <xref
+       linkend="sec-depl-maintenance-bootstrap-compute-plane"/>.
```

Comment:
I think this is targetting the wrong section. AFAIU the idea is to reference the section about recovering galera somehow: https://github.com/SUSE-Cloud/doc-cloud/blob/maintenance/cloud_9/xml/soc-operations-maintenance.xml#L1338 but obviously that is not available in Cloud 8 (yet?).

However adding that note about that here is also misleading I think. Because the step outline above won't really work if the database cluster is down.

How about just adding a note at the beginning of the chapter, that in order to run these update steps the database cluster needs to be up and healthy.